src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
-}
+  const inverse = {};
+  const values = Object.values(robot);
+
+  for (let i = 0; i < values.length; i++) {
```

Comment:
u can just use for in loop for robot & and do all u need instead of creating 3 loops